test/Microsoft.Azure.SignalR.AspNet.Tests/Infrastructure/VerifyNoErrorsScope.cs

```diff
@@ -32,9 +32,13 @@ public void Dispose()
 
             var results = _sink.GetLogs().Where(w => w.Write.LogLevel >= LogLevel.Error).ToList();
 
-            if (_expectedErrorsFilter != null)
+            if (_expectedErrorsMatch != null)
             {
-                results = results.Where(w => !_expectedErrorsFilter(w.Write)).ToList();
+                if (results.Where(w => _expectedErrorsMatch(w.Write)).ToList().Count == 0)
+                {
+                    throw new Exception("Fail to match expected error(s).");
+                }
+                results = results.Where(w => !_expectedErrorsMatch(w.Write)).ToList();
```

Comment:
`ToList()` twice

Reply:
Where returns Enumerable and need to convert to List for further no error check.